front-end/src/app/layout/layout.component.html

```diff
@@ -32,22 +32,16 @@
 
   <!-- Begin No Sidebar section -->
   <div *ngIf="!layoutControls.showSidebar" id="content-offset">
-    <div class="container">
```

Comment:
It looks like this change is having an effect on the header on wide views.  I don't think there is supposed to be a change to the header as a result of this ticket

As implemented:
![image](https://github.com/user-attachments/assets/13ce8b08-4d53-4677-a341-52e96adb9a3d)

On Dev:
![image](https://github.com/user-attachments/assets/08bd59de-e5b7-412a-8143-77e6ea513f31)